samples.yml

```diff
@@ -232,6 +232,8 @@ mc_summer22_MINIAODv4:
         - /QCD_PT-2400to3200_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
         - /QCD_PT-2400to3200_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5_ext1-v2/MINIAODSIM
         - /QCD_PT-3200_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+        # - /QCD_PT-3200_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv3-124X_mcRun3_2022_realistic_v12_ext1-v2/MINIAODSIM
```

Comment:
This is commented out but kept as a placeholder like: https://github.com/cms-btv-pog/btvnano-prod/blob/ed9d92b39978067ede248113541fa3f36e0baff2/samples.yml#L207

I am not sure if this would also be produced in future? If not, it can be removed

Reply:
ah ok, sorry about the confusion indeed they are produced 
/QCD_PT-3200_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5_ext1-v2/MINIAODSIM
![image](https://github.com/cms-btv-pog/btvnano-prod/assets/20898477/ecb5cc0d-a512-4678-b444-618f17e097b4)
IMO should be good to go :p could you please also submit this !